packages/node_modules/@node-red/nodes/locales/en-US/messages.json

```diff
@@ -350,12 +350,37 @@
             "retain": "Retain",
             "clientid": "Client ID",
             "port": "Port",
-            "keepalive": "Keep alive time (s)",
+            "keepalive": "Keepalive time(s)",
             "cleansession": "Use clean session",
             "use-tls": "Enable secure (SSL/TLS) connection",
             "tls-config":"TLS Configuration",
             "verify-server-cert":"Verify server certificate",
-            "compatmode": "Use legacy MQTT 3.1 support"
+            "compatmode": "Use legacy MQTT 3.1 support",
+            "userProperties": "User Properties",
+            "subscriptionIdentifier": "Subscription ID",
+            "flags": "Flags",
+            "nl": "No Local",
+            "rap": "Retain as Published",
+            "rh": "Retain Handling",
+            "rh0": "0, Always send retained messages",
+            "rh1": "1, Send retained messages for new subscription",
```

Comment:
I'm not sure it makes sense to expose the 'send on new sub' option.  We don't let the user dynamically subscribe, so the only time they would have a resubscribe is if they had two identically configured nodes.

I think this option should be simplified to a binary "Send retained messages" options.

Reply:
This was purely to follow the oasis specification Nick. It does work as described. But as you have 1st hand knowledge on the design of both node-red and mqtt, I am of course happy to go with your recommendation.